third_party/intel/lib/Utils/Mangling.cpp

```diff
@@ -20,7 +20,7 @@ std::string getTypeMangling(Type ty, bool isUnsigned) {
       .Case([isUnsigned](IntegerType ty) -> std::string {
         switch (ty.getWidth()) {
         case 8:
-          return "c";
+          return isUnsigned ? "c" : "h";
```

Comment:
```suggestion
          return isUnsigned ? "h" : "c";
```

`c` is for char, `h` for unsigned char (see https://itanium-cxx-abi.github.io/cxx-abi/abi.html#mangling-builtin)

Reply:
done